doc/getting-started/src/tutorial-example-for-running-energyplus/instructions.tex

```diff
@@ -28,7 +28,7 @@ \subsection{Exercise 1A. Run Pre-Defined Building with no Windows}\label{exercis
 \item
   Outdoor Dry Bulb -- is being reported (so you can compare to outside temperature)
 \item
-  The meter for the heating in the facility - DistrictHeating:Facility -- is being reported. Facility is the entire building.
+  The meter for the heating in the facility - DistrictHeatingWater:Facility -- is being reported. Facility is the entire building.
```

Comment:
I didn't know this was wrong. What about this?

![image](https://github.com/user-attachments/assets/ccb2529e-605f-44a8-ad5d-6c6432da9fd1)

Reply:
Oh never mind, that's a comment.